### ISSUE TITLE
Icon request: presenter-pointer

### ISSUE THREAD

**Issue by DieterGust** [FortAwesome/Font-Awesome#15238]
**What can it be used for?**
to visualize presentation steps/tasks

**Example image (optional)**
![laser pointer](https://user-images.githubusercontent.com/33634168/60680677-88d85600-9e8c-11e9-8867-2a36f3eea38c.png)

**Icon request checklist**
- [x ] This is a single icon or matched pair (Ex: `lock` / `unlock`)
- [x ] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
- [x ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [x] I have included at least one example image if it could be helpful (optional)
- [ x] The request is for a concrete object (it's harder to make an icon to represent happiness, it's easier to make a smiley face. ☺)

**Comment by trymhemo** [comment]
Hey! I like your idea. I've come up with something similar.
I've shared the source code of the SVG i made in this [Icon request](https://github.com/FortAwesome/Font-Awesome/issues/18123)
Until font-awesome implements anything, I thought that my SVG would be of great interest for you and suitable for your intended use. If you dont mind I would be much appreaciated with you liked my post, and commented it with whatever change you think it needs. I would be happy to email you a revised version for you and your project.

Best regards
Trym Gonzales

**Comment by tagliala** [comment]
Originally posted in #18123

> ### TL;DR:
> _I made an icon that you were missing. I am happy to share the finished product with you. **I have attached the svg soruce code** in this Icon request, but keep in mind it's just the first draft/prototype. It has been though just a few user-tests, but we've landed on something intuive that certainly is of value to you. To get in contact please go through discord: Tryym#7504 , I don't respond in here
> 
> ### Context
> My job is working with educational and collaborating services. I have never failed to find an icon I am looking for when I am using font-awesome as my time as a developer. Until yesterday. I was trying to look for an icon in your collection to represent a laser-pointer/laser-pen. Probably not a surprise, but it's supposed to be used for a button that enables a pointer. More specifically, to be used when a presenter wants to point at specific parts of a documents for an audience instead of drawing notes with a brush. There are more and more companies that offer internet studies and companies that now have adapted to online organization. Because of this, there will soon or already now be a demand for icons that accommodate a workday in office jobs from home. Especially different tools we had never thought of translating to virtual counterparts. In this case, a laser pointer.
> Below, you can find the source svg code for the icon I have made. It's good enought to use, but please contact me regaring this before putting anythign to use. Next week me and my group are gonna review and test usability between different versions and their utility in small and vague contrasts...ect We plan to make an outlined version as well.
> As mentioned in TL;DR, please contact me on my discord: Tryym#7504. I don't respond in here
> 
> ### PNG render of the pointer in it's current state
> ![image](https://user-images.githubusercontent.com/43746246/127630871-240f8c5d-180b-4765-b33d-d49f6a211c23.png)
> 
> ### SVG code for said laser pointer icon
> `<svg xmlns="http://www.w3.org/2000/svg" width="189.236" height="260.125" viewBox="0 0 189.236 260.125"> <g id="Laser" transform="translate(-42.082 -82.618)"> <path id="Subtraction_1" data-name="Subtraction 1" d="M-3360-691h-100a10.011,10.011,0,0,1-10-10v-26a10.011,10.011,0,0,1,10-10h100a10.011,10.011,0,0,1,10,10v26A10.011,10.011,0,0,1-3360-691Zm-15.5-31a8.51,8.51,0,0,0-8.5,8.5,8.51,8.51,0,0,0,8.5,8.5,8.51,8.51,0,0,0,8.5-8.5A8.509,8.509,0,0,0-3375.5-722Z" transform="translate(2415.343 -2316.865) rotate(-60)" fill="#888d9c"/> <path id="Rectangle_2454" data-name="Rectangle 2454" d="M0,0H77a5,5,0,0,1,5,5V6a5,5,0,0,1-5,5H0a0,0,0,0,1,0,0V0A0,0,0,0,1,0,0Z" transform="translate(121.304 216.526) rotate(-60)" fill="#888d9c"/> <path id="Subtraction_2" data-name="Subtraction 2" d="M-3388-681a27.825,27.825,0,0,1-16.049-5.053A27.953,27.953,0,0,1-3414.161-699H-3393a10.011,10.011,0,0,0,10-10,10.011,10.011,0,0,0-10-10h-21.161a27.953,27.953,0,0,1,10.113-12.947A27.825,27.825,0,0,1-3388-737a27.826,27.826,0,0,1,10.9,2.2,27.914,27.914,0,0,1,8.9,6,27.91,27.91,0,0,1,6,8.9A27.82,27.82,0,0,1-3360-709a27.82,27.82,0,0,1-2.2,10.9,27.91,27.91,0,0,1-6,8.9,27.914,27.914,0,0,1-8.9,6A27.826,27.826,0,0,1-3388-681Z" transform="translate(2475.513 -2431.082) rotate(-60)" fill="#888d9c"/> <rect id="Rectangle_2457" data-name="Rectangle 2457" width="9" height="44" rx="4" transform="translate(107.638 173.443) rotate(-108)" fill="#888d9c"/> <rect id="Rectangle_2458" data-name="Rectangle 2458" width="9" height="44" rx="4" transform="matrix(-0.978, 0.208, -0.208, -0.978, 183.778, 210.448)" fill="#888d9c"/> <rect id="Rectangle_2459" data-name="Rectangle 2459" width="9" height="44" rx="4" transform="translate(189.472 145.634) rotate(-108)" fill="#888d9c"/> <rect id="Rectangle_2460" data-name="Rectangle 2460" width="9" height="44" rx="4" transform="matrix(-0.978, 0.208, -0.208, -0.978, 166.915, 125.657)" fill="#888d9c"/> <rect id="Rectangle_2461" data-name="Rectangle 2461" width="9" height="44" rx="4" transform="translate(182.599 128.361) rotate(-150)" fill="#888d9c"/> <rect id="Rectangle_2462" data-name="Rectangle 2462" width="9" height="33.5" rx="4" transform="translate(150.582 131.816) rotate(120)" fill="#888d9c"/> <rect id="Rectangle_2464" data-name="Rectangle 2464" width="9" height="33.5" rx="4" transform="translate(219.43 171.566) rotate(120)" fill="#888d9c"/> <rect id="Rectangle_2465" data-name="Rectangle 2465" width="9" height="33.5" rx="4" transform="translate(155.603 141.643) rotate(95)" fill="#888d9c"/> <rect id="Rectangle_2466" data-name="Rectangle 2466" width="9" height="33.5" rx="4" transform="translate(164.135 134.47) rotate(145)" fill="#888d9c"/> <rect id="Rectangle_2467" data-name="Rectangle 2467" width="9" height="33.5" rx="4" transform="translate(181.395 153.854) rotate(-85)" fill="#888d9c"/> <rect id="Rectangle_2468" data-name="Rectangle 2468" width="9" height="33.5" rx="4" transform="translate(197.747 182.323) rotate(145)" fill="#888d9c"/> <rect id="Rectangle_2469" data-name="Rectangle 2469" width="9" height="33.5" rx="4" transform="matrix(-0.993, -0.122, 0.122, -0.993, 173.59, 135.469)" fill="#888d9c"/> <rect id="Rectangle_2470" data-name="Rectangle 2470" width="9" height="33.5" rx="4" transform="matrix(0.602, 0.799, -0.799, 0.602, 200.344, 115.309)" fill="#888d9c"/> </g> </svg>`
> 
> **Icon request checklist**
> 
> * [x ] This is a single icon or matched pair (Ex: `lock` / `unlock`)
> * [x ] The title starts with "Icon request: " and is followed by the requested icon name (Ex: `Icon request: magical-unicorn`)
> * [x ] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues)
>   **_I've found somebody requesting a "presenter pointer", but their idea it just not intuitive enough in my opinion..._**
> * [x ] I have included at least one example image if it could be helpful (optional)
> * [x ] The request is for a concrete object

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’re closing this issue because we think it might be too niche, or just not a good fit for Font Awesome.

But if you feel really strongly about it, feel free to make your case by submitting a new issue.
